test/engine-luatest/pagination_test.lua

```diff
@@ -6,6 +6,8 @@ local tree_g = t.group('Tree index tests', {
     {engine = 'memtx', disable_ffi = false},
     -- Vinyl doesn't use FFI.
     {engine = 'vinyl', disable_ffi = false},
+    -- Read view does not use FFI.
+    {engine = 'read_view', disable_ffi = false},
```

Comment:
`read_view` isn't an engine. Technically the vinyl engine may support read views as well.

Reply:
Added a separate option `is_rv`